pkg/controller/controller.go

```diff
@@ -299,6 +299,10 @@ func (c *Controller) initActionSetStatus(as *crv1alpha1.ActionSet) {
 	}
 	if err != nil {
 		as.Status.State = crv1alpha1.StateFailed
+		as.Status.Error = crv1alpha1.Error{
```

Comment:
We should be able to simplify this because we don't use a pointer:
```suggestion
		as.Status.Error.Message = err.Error()
```